core/src/validator.rs

```diff
@@ -1502,6 +1504,7 @@ impl Validator {
                 retransmit: node.sockets.retransmit_sockets,
                 fetch: node.sockets.tvu,
                 ancestor_hashes_requests: node.sockets.ancestor_hashes_requests,
+                alpenglow: node.sockets.alpenglow,
```

Comment:
could we get rid of the `enable_all2all_tests` flag and just override the `alpenglow` socket to `None` based on `cluster_type`?